app/assets/stylesheets/components/common/_global_subnav.scss

```diff
@@ -152,11 +156,10 @@ $global-nav-transition-duration: 300ms !default; // Defined in _global_nav.scss
 }
 
 .global-subnav__tool-category {
-  display: none;
+  display: table-cell;
 
```

Comment:
Not sure about table-cell here. We have used flexbox on desktop and table-cell as a fallback on non-compliant browsers. Not a big deal maybe: depends on what, erm, design we're working to I suppose.

Reply:
Updated per our conversation.